### PR TITLE
Reviewed and enhanced (template, tasks). Allow multiple inventory variables and more.

### PR DESCRIPTION
**Pull request by ypid**
- Allow global, group_vars and host_vars configuration.
- Fixed and optimized template. (includedir without # in front it does not work on Debian jessie)
- Allow more flexible configuration like handing over a command list instead of a string (strings are still supported).
- Ubuntu and Linuxmint are also supported (untested but very likely).
- Changes should be fully backwards compatible.

Changes where made after review of all sudo roles on Ansible Galaxy.
This role (picotrading.sudo) best fits my purpose.

Other relevant roles are:
- TalLannder.sudo: https://github.com/TalLannder/ansibleRole-Sudo/pull/1
- vozerov.sudo: https://github.com/vozerov/ansible-sudo/pull/1
- weareinteractive.sudo: https://github.com/weareinteractive/ansible-sudo/pull/4

### CODE REVIEW

**Comment by jtyr** [comment]
Thanks for the review. Unfortunately most of the changes are only cosmetic. The main point of this pull request (be able to add configuration for host and group) can be solved by the construct described here:

https://github.com/picotrading/ansible-sysctl/pull/1

**Comment by ypid** [comment]
Might be still useful:
- Allow more flexible configuration like handing over a command list instead of a string (strings are still supported).
- Ubuntu and Linuxmint are also supported (untested but very likely).

**Comment by jtyr** [comment]
Handing variables over a list is limited to two more variables in your pull request. Therefore it makes it quite unlikely that somebody would use a list instead of the string.

Ubuntu and Linixmint are derivatives of Debian so there is no need to mention extra distributions. The code should just work.

File: defaults/main.yaml [hunk 1]
```diff
@@ -1,32 +1,38 @@
...
+#     - lisa
+
+sudo_user_alias_group: {}
+sudo_user_alias_host: {}
 
 # Default command aliases
 sudo_cmd_alias: {}
 #sudo_cmd_alias:
-#  SOFTWARE
+#  SOFTWARE:
```
Comment (jtyr) [inline defaults/main.yaml:35]: This line is actually the only acceptable change.

**Comment by ypid** [comment]
All right. I guess its easier when you make this single change directly I guess :wink:.

**Pull request closed without merging**